src/Build/Microsoft.Build.csproj

```diff
@@ -27,6 +27,11 @@
     <NoWarn>$(NoWarn);NU5104</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <ApiCompatPreserveUnnecessarySuppressions>true</ApiCompatPreserveUnnecessarySuppressions>
```

Comment:
These are not ideal, but the check is currently failing in a weird way where it is complaining about the comments that we have on the xml files.